abilities/delhi/tower-of-victory-aura-2.json

```diff
@@ -0,0 +1,42 @@
+{
+  "id": "tower-of-victory-aura-2",
+  "baseId": "tower-of-victory-aura",
+  "type": "ability",
+  "name": "Tower of Victory Aura",
+  "pbgid": 200847,
+  "attribName": "tower_of_victory_aura_sul",
+  "age": 2,
+  "civs": [
+    "de"
+  ],
+  "description": "Melee and ranged infantry who move near this Landmark permanently gain +20% attack speed.",
+  "classes": [],
+  "displayClasses": "",
+  "costs": {
+    "food": 0,
+    "wood": 0,
+    "stone": 0,
+    "gold": 0,
+    "total": 0,
+    "popcap": 0
+  },
+  "producedBy": [],
+  "icon": "https://data.aoe4world.com/images/abilities/tower-of-victory-aura-1.png",
+  "active": "always",
+  "auraRange": 7.5,
+  "effects": [
+    {
+      "property": "attackSpeed",
+      "select": {
+        "class": [
+          [
+            "infantry"
+          ]
+        ]
+      },
+      "effect": "multiply",
+      "value": 0.83,
+      "type": "ability"
```

Comment:
I would argue this is passive or influence, as it is a permanent effect

Reply:
Below is what I have so far on this topic:

So far, every ability has **"type":"ability"** and every abilities effects/modifiers has **"type":"ability"**

ability has an **active** property with three possible values:

- always
- manual
- toggle

every ability with **"active":"manual"** should and currently does have a **duration** in the effects


some ability with **"active":"always"** has **duration**:

- inspired-warriors-3
- lancer-charge-bonus-damage-2
- spirit-way-4
- huntress-stealth-4
- saints-blessing-3

Recall that **"active"** is obtained directly from the **/abilities** file path or a workaround if in **/info/buff_info**

Reply:
Oke, clear reasoning. LGTM as it is